ops/genesis/config.go

```diff
@@ -94,6 +95,17 @@ func GeneratePredeployConfig(config *GenesisConfig, block *types.Block) predeplo
 				"PORTAL_ADDRESS": {ProxyValue: *predeploys.Predeploys["L2Portal"]},
 			},
 		},
+		"L2BaseFeeVault": {
+			Proxied:     true,
+			Initializer: "initialize",
+			Storages: map[string]predeploys.StorageConfig{
+				"_initialized":        {ProxyValue: InitializedValue, ImplValue: MaxInitializedValue},
+				"_initializing":       {ProxyValue: false, ImplValue: false},
+				"_owner":              {ProxyValue: config.L2PredeployOwner},
+				"recipient":           {ProxyValue: config.L2PredeployOwner},
```

Comment:
should this be `config.L2BaseFeeRecipient`?

Reply:
recipient is the address that accumulated vault fees can be withdrawn to. should be more clear with the rename to `L2BaseFeeRecipient`